tests/state/test_state_transition.py

```diff
@@ -29,3 +38,89 @@ def test_validate_attestation_bitfield():
 
 def test_validate_attestation_aggregate_sig():
     pass
+
+
+@pytest.mark.parametrize(
+    (
+        'cycle_length,last_state_recalc,'
+        'last_justified_slot,justified_streak,last_finalized_slot,'
+        'crystallized_state_total_deposits,block_vote_cache_total_deposits,'
+        'result_last_state_recalc,result_justified_streak, result_last_finalized_slot'
+    ),
+    [
+        (64, 0, 0, 0, 0, 3, 2, 64, 0+64, 0),  # 2/3 attestations
```

Comment:
formatting: `0+64` -> `0 + 64` (same in following lines)

Reply:
I find it's harder to read:
```python
    [
        (64, 0, 0, 0, 0, 3, 2, 64, 0 + 64, 0),  # 2/3 attestations
        (64, 0, 0, 0, 0, 3, 1, 64, 0 + 0, 0),   # 1/3 attestations
        (64, 128, 128, 64, 0, 3, 2, 128 + 64, 64 + 64, 127 - 64 - 1),  # 2/3 attestations, last_finalized_slot = slot - cycle_length - 1
        (64, 128, 128, 128, 128, 3, 2, 128 + 64, 128 + 64, 128),  # 2/3 attestations, last_finalized_slot = last_finalized_slot
    ],
```
Not the same case, but similar to something like https://www.python.org/dev/peps/pep-0008/#other-recommendations
`hypot2 = x*x + y*y` is better than `hypot2 = x * x + y * y` sometimes.